interface/controller/student_controller.go

```diff
@@ -0,0 +1,58 @@
+package controller
+
+import (
+	"encoding/csv"
+	"errors"
+	"golang-bootcamp-2020/domain/model"
+	"io"
+	"log"
+	"os"
+	"strconv"
+)
+
+type StudentController interface {
+	GetStudents() []model.Student
+}
+
+func GetStudents() []model.Student {
+	csvFile, err := os.Open("infrastructure/datastore/dataFile.csv")
```

Comment:
This could be improved later:
Move the path file to a config file

Reply:
Done